### DISCUSSION TITLE
能请教一个问题吗

### DISCUSSION THREAD

**Discussion by letterk** [821938089/MyNovelReader#27] [General]
自从起点防盗以后, 我的阅读体验直线下降. 需要忍受不喜欢的背景和字体, 也不能更改排版
我喜欢滚动加载下一章的方式
但是起点的滚动加载, 不会更新标题, 不会更新地址栏, 不会加入历史记录. 
我想通过油猴脚本来解决这三个需求
但是我卡在了一个地方, 我不知道该如何确定当前可视区域"实时"对应的哪个元素
搜索引擎里, 只是找到很多判断某个已知元素是否在可视区域的例子. 

如果你有时间的话, 能给我一点引导吗? 谢谢

**Comment by 821938089** [comment]
这个问题我反馈过，新版的阅读页已经修复了（除了标题更新），标题问题我反馈的时候没想到，漏了，现在反馈调查问卷入口没了，不知道怎么反馈了
你可以装这个脚本解决：https://github.com/821938089/QidianFix

**Reply by letterk** [reply]
我去兔小巢反馈了标题更新的问题. 哪里有太多喷起点的反馈, 不知道有没有人看
